src/lib/structures/music/MusicHandler.ts

```diff
@@ -102,7 +101,7 @@ export class MusicHandler {
 	public async setVolume(volume: number, context: MusicHandlerRequestContext | null = null) {
 		if (volume <= 0) throw this.guild.language.tget('MUSICMANAGER_SETVOLUME_SILENT');
 		if (volume > 200) throw this.guild.language.tget('MUSICMANAGER_SETVOLUME_LOUD');
-		await this.player.setVolume(volume);
+		await this.player!.volume(volume);
 		this.client.emit(Events.MusicSongVolumeUpdate, this, this.volume, volume, context);
```

Comment:
idk whether this is helpful or not but Player class has a volume event and a seek event

Reply:
This is to change the volume, not listen to when volume changes. And any further processing is handled by our own event, `musicSongVolumeUpdate` which is where it is because of the websocket to the dashboard.